es2csv.py

```diff
@@ -267,6 +267,11 @@ def main():
     p.add_argument('-e', '--meta_fields', dest='meta_fields', action='store_true', help='Add meta-fields in output.')
     p.add_argument('-v', '--version', action='version', version='%(prog)s ' + __version__, help='Show version and exit.')
     p.add_argument('--debug', dest='debug_mode', action='store_true', help='Debug mode on.')
+    p.add_argument('--verify-certs', dest='verify_certs', action='store_true', help='Verify SSL Certs (default is False)')
+    p.add_argument('--ca-certs', dest='ca_certs', default='', type=str, help='Location of CA cert file for Cert verification')
+    p.add_argument('--client-cert', dest='client_cert', default='', type=str, help='Location of Client Auth cert')
+    p.add_argument('--client-key', dest='client_key', default='', type=str, help='Location of Client Cert Key')
+    p.add_argument('--use-ssl', dest='use_ssl', action='store_true', help='Use SSL in the Elasticsearch connection')
```

Comment:
As we are using url it automatically switched to true when you pass https url.
https://github.com/elastic/elasticsearch-py/blob/5.0/elasticsearch/client/__init__.py#L46